vendor/knative.dev/pkg/tracing/zipkin.go

```diff
@@ -32,7 +32,7 @@ type ZipkinReporterFactory func(*config.Config) (zipkinreporter.Reporter, error)
 //  - WithExporter() in production code
 //  - testing/FakeZipkinExporter() in test code.
 func WithZipkinExporter(reporterFact ZipkinReporterFactory, endpoint *zipkinmodel.Endpoint) ConfigOption {
-	return func(cfg *config.Config) {
+	return func(cfg *config.Config) error {
```

Comment:
Same as the previous comments. I think it should be an update to knative pkg repo.

Reply:
Absolutely, https://github.com/knative/pkg/pull/657 is making the change in pkg.